tests/model/ServiceLocator.php

```diff
@@ -38,10 +38,25 @@ public static function getCacheStorage()
 
 
 	/** @return Nette\Database\Context */
-	public static function getDbContext()
+	public static function getDbContext(): Nette\Database\Context
 	{
 		if (self::$dbContext === NULL) {
-			$connection = new Nette\Database\Connection('mysql:host=127.0.0.1;dbname=yetorm_test', 'root', '');
+			$configFile = __DIR__ . '/../config/local.neon';
```

Comment:
Please remove this if/else logic since it has nothing to do with this PR.

Reply:
Do you think its good idea to send separete PR for this? Its hard set up automatic testing when local DB has password. Can be solved be optional config.neon.

Reply:
Otherwise resolved.